user/api.go

```diff
@@ -46,9 +48,10 @@ var (
 )
 
 type (
+	// Api struct used by shoreline server components
 	Api struct {
 		Store        Storage
-		ApiConfig    ApiConfig
+		ApiConfig    *ApiConfig
```

Comment:
To avoid some useless object copy, and to be coherent with the New() methods.
Since they generally return a pointer on an object (at least considering others compiled language, new return a pointer)